src/Raven.Server/Dashboard/Cluster/Notifications/NotificationsSummaryNotificationSender.cs

```diff
@@ -0,0 +1,85 @@
+    private NotificationsSummaryRequestConfig _notificationsSummaryRequestConfig;
+    
+    public NotificationsSummaryNotificationSender(int widgetId, DatabasesInfoRetriever databasesInfoRetriever, ConnectedWatcher watcher, BlittableJsonReaderObject configuration, CancellationToken shutdown) : base(widgetId, watcher, shutdown)
+    {
+        _databasesInfoRetriever = databasesInfoRetriever;
+        _notificationsSummaryRequestConfig = JsonDeserializationServer.NotificationsSummaryRequestConfig(configuration);
+    }
+
+    protected override TimeSpan NotificationInterval { get; } = TimeSpan.FromSeconds(5);
+    
+    protected override AbstractClusterDashboardNotification CreateNotification()
+    {
+        var databasesNotificationsSummary = _databasesInfoRetriever.GetDatabaseNotificationsSummary();
+
+        var notificationsSummaryPayload = new NotificationsSummaryPayload();
+
+        foreach (var item in databasesNotificationsSummary.Items)
+        {
+            var databaseNotificationsSummary = new DatabaseNotificationsSummary()
+            {
+                DatabaseName = item.DatabaseName
+            };
+
+            foreach (var notificationCountsKvp in item.NotificationCounts.ByType)
+            {
+                var notificationType = notificationCountsKvp.Key;
+
+                var typeSettings = notificationType switch
+                {
+                    NotificationType.AlertRaised => _notificationsSummaryRequestConfig.Alerts,
+                    NotificationType.PerformanceHint => _notificationsSummaryRequestConfig.PerformanceHints,
+                    _ => null
+                };
+                
+                if (typeSettings == null || typeSettings.IsEnabled == false)
+                    continue;
+                
+                foreach (var notificationReasonCount in notificationCountsKvp.Value.ByReason)
+                {
+                    if (typeSettings.Reasons.Count > 0 && typeSettings.Reasons.Contains(notificationReasonCount.Key) == false)
+                        continue;
+                    
+                    var notificationSummaryItem = new NotificationSummaryItem
+                    {
+                        Reason = notificationReasonCount.Key,
+                        Count = notificationReasonCount.Value
+                    };
+
+                    switch (notificationType)
+                    {
+                        case NotificationType.AlertRaised:
+                            databaseNotificationsSummary.Alerts.Add(notificationSummaryItem);
+                            break;
+                        case NotificationType.PerformanceHint:
+                            databaseNotificationsSummary.PerformanceHints.Add(notificationSummaryItem);
+                            break;
+                        default:
+                            throw new Exception($"Unsupported {nameof(NotificationType)}: {notificationType}");
```

Comment:
`NotSupportedException`? Would that break the studio? Let's ensure we have a test that will fail in case we add new type